tests/e2e/util.go

```diff
@@ -1632,13 +1636,72 @@ func DeleteStatefulPodAtIndex(client clientset.Interface, index int, ss *apps.St
 
 }
 
+//getClusterComputeResource returns the clusterComputeResource and vSANClient
+func getClusterComputeResource(ctx context.Context, vs *vSphere) ([]*object.ClusterComputeResource, *VsanClient) {
+	var err error
+	if clusterComputeResource == nil {
+		clusterComputeResource, vsanHealthClient, err = getClusterName(ctx, vs)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	}
+	return clusterComputeResource, vsanHealthClient
+}
+
+//findIP returns the IP from the input string
+func findIP(input string) string {
+	numBlock := "(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])"
+	regexPattern := numBlock + "\\." + numBlock + "\\." + numBlock + "\\." + numBlock
+
+	regEx := regexp.MustCompile(regexPattern)
+	return regEx.FindString(input)
+}
+
+//getHosts returns list of hosts and it takes clusterComputeResource as input
+//This method is used by WCP and GC tests
+func getHosts(ctx context.Context, clusterComputeResource []*object.ClusterComputeResource) []*object.HostSystem {
+	var err error
+	if hosts == nil {
+		computeCluster := os.Getenv(envComputerClusterName)
+		if computeCluster == "" {
+			if guestCluster {
+				computeCluster = "compute-cluster"
+			} else if supervisorCluster {
+				computeCluster = "wcp-app-platform-sanity-cluster"
+			}
+			framework.Logf("Default cluster is chosen for test")
+		}
+		for _, cluster := range clusterComputeResource {
+			framework.Logf("clusterComputeResource %v", clusterComputeResource)
+			if strings.Contains(cluster.Name(), computeCluster) {
+				hosts, err = cluster.Hosts(ctx)
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+				framework.Logf("host information %v", hosts)
+			}
+		}
+	}
+	gomega.Expect(err).NotTo(gomega.BeNil())
+	return hosts
+}
+
+//checkAllHostStatuswithwait will check and wait till the host is reachable
+func checkAllHostStatusWithWait(ctx context.Context, vs *vSphere) {
+	clusterComputeResource, vsanHealthClient = getClusterComputeResource(ctx, vs)
+	hosts = getHosts(ctx, clusterComputeResource)
+	framework.Logf("host information %v", hosts)
```

Comment:
you are logging this in `getHosts()` as well, remove it from on of the places
I recommend removing it from `getHosts()`

Reply:
done